python/cuml/_version.py

```diff
@@ -16,6 +16,11 @@
 import importlib.resources
 
 __version__ = (
-    importlib.resources.files("cuml").joinpath("VERSION").read_text().strip()
+    importlib.resources.files(__package__).joinpath("VERSION").read_text().strip()
 )
-__git_commit__ = ""
+try:
+    __git_commit__ = importlib.resources.files(__package__).joinpath("GIT_COMMIT").read_text().strip()
+except FileNotFoundError:
+    __git_commit__ = ""
+
+__all__ = ["__version__", "__git_commit__"]
```

Comment:
Actually, before we merge this, could you please:

* add a unit test checking these constants (https://github.com/rapidsai/cugraph/pull/4393#discussion_r1619516285)
* alphabetize this `__all__` list? (https://github.com/rapidsai/cugraph/pull/4393#discussion_r1619516809)